src/UnitConverter/Unit/Time/Month.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2017 Jordan Brauer <jbrauer.inc@gmail.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Time;
+
+/**
+ * Month unit data class.
+ *
+ * @version 1.0.0
+ * @since 1.0.0
+ * @author Teun Willems
+ */
+class Month extends TimeUnit
+{
+    protected function configure () : void
+    {
+        $this
+            ->setName("month")
+
+            ->setSymbol("month")
+
+            ->setUnits(2678400)
```

Comment:
Where did you get this number? This isn't what I found from Googles' converter and other random converters. They all land between 2,628,000 and 2,629,000.

1 month === 2.682e+6 seconds
2.682e+6 = (2.628 * pow(10,6))
= 2,628,000 seconds